### PR TITLE
Milestone4

### PR DESCRIPTION
**Pull request by Ambrosegithub**
## Added contact sections.

- [ ] Use HTML5 semantic tags
- [ ] Use css3
- [ ] Use git
- [ ] Use linter for error check
- [ ] [Tracy](https://github.com/Elfin-git
) contributed greatly
- [ ] [Aziz](https://github.com/AzizRAA
) contribute greatly

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -150,5 +150,43 @@ <h5>Skills</h5>
...
+        <input class="input-field" type="text" placeholder="Enter your name" id="user_name" required maxlength="30">
+      </label>
+
+     <label for="user_email">E-mail<input type="email" id="user_email" placeholder="Enter your email " required maxlength="30">
+      </label>
+      </div>
+       <textarea name="text" id="user-email" cols="34" rows="10" placeholder="Enter your message here" required maxlength="500"></textarea>
+       <button type="submit">Start Colaboration</button>
+       <img  src="./images/Vector1.png" id="btn-img"  alt="">
+    </form>
```
Comment (rustam-tolipov) [inline index.html]: - [ ] Good job so far 👏🏼🙌🏼. The form is working correctly 💯 but it needs some changes in design. Let's change `.contact-form` background to `#F9F9FA`.

File: index.html [hunk 2]
```diff
@@ -150,5 +150,43 @@ <h5>Skills</h5>
       </div>
     </div>
   </section>
+  <section class="contact-form">
+    <div id="bg-contact-img1"> <img src="./images/image3.png" alt=""></div>
+  <div id="bg-contact-img"></div>
+  <h2>Get Started</h2>
```
Comment (rustam-tolipov) [inline index.html]: - [ ] Please increase the heading font size and change the font family to `Playfair Display`.

Your website
image

Original design
image

File: index.html [hunk 3]
```diff
@@ -150,5 +150,43 @@ <h5>Skills</h5>
...
+  <div><p>Lorem ipsum, dolor sit amet consectetur adipisicing 
+    elit. Sint placeat dolorum perspiciatis rerum 
+    molestiae dicta aperiam libero quod officiis odio.</p></div>
+    <form action="https://formspree.io/f/mpzbbknz" method="POST">
+        <div class="form-dir">
+      <label for="user_name" id="name"> <span>Name</span>
```
Comment (rustam-tolipov) [inline index.html]: - [ ] Input fields do not look like the original design. I'd suggest adding some padding and changing the border color to `#DFDFE0`

File: index.html [hunk 4]
```diff
@@ -150,5 +150,43 @@ <h5>Skills</h5>
...
+        <div class="form-dir">
+      <label for="user_name" id="name"> <span>Name</span> 
+        <input class="input-field" type="text" placeholder="Enter your name" id="user_name" required maxlength="30">
+      </label>
+
+     <label for="user_email">E-mail<input type="email" id="user_email" placeholder="Enter your email " required maxlength="30">
+      </label>
+      </div>
+       <textarea name="text" id="user-email" cols="34" rows="10" placeholder="Enter your message here" required maxlength="500"></textarea>
+       <button type="submit">Start Colaboration</button>
```
Comment (rustam-tolipov) [inline index.html]: - [ ] Buttons should have blue color on normal position and white color on click. I think you chose the opposite 😉. Please fix this issue. You can refer to this image
image

File: index.html [hunk 5]
```diff
@@ -150,5 +150,43 @@ <h5>Skills</h5>
...
+      <div id="footer-img-right">
+          <img src="./images/ic_stackoverflow.png" alt="">
+          <img src= "./images/Vector1.png" alt="">
+          <img src="./images/Vector2.png" alt="">
+          <img src="./images/Vector5.png" alt="">
+          <img src="./images/Vector3.png" alt="">
+          <img src="./images/Vector4.png" alt="">
+    </div>
+         </div>
+</footer>
```
Comment (rustam-tolipov) [inline index.html]: - [ ] There is some additional space below the footer. You can fix it by removing `margin-bottom: 10px` from `<footer>` tag 👍🏼

Screenshot
image

File: index.html [hunk 6]
```diff
@@ -150,5 +150,43 @@ <h5>Skills</h5>
...
+       <img  src="./images/Vector1.png" id="btn-img"  alt="">
+    </form>
+</section>
+<footer>
+  <div id="resume-img"> 
+      <h6>Get my resume </h6>
+    <img  id="cross-img" src="./images/ic_link_teal.png" alt="">
+  </div>
+  <div id="footer-img">
+      <div id="footer-img-left">Get my cv<img src="./images/ic-link_teal.png" alt=""></div>
```
Comment (rustam-tolipov) [inline index.html]: - [ ] Finally, add some space on top of the 'get cv' link. Good luck with that 🙌🏼

**Review by rustam-tolipov — CHANGES_REQUESTED** [review]
Hi @Ambrosegithub , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
_Your project is almost complete!_


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (Ambrosegithub) [reply index.html]: Done

File: index.html (hunk 3 above)
Comment (Ambrosegithub) [reply index.html]: Done

File: index.html (hunk 4 above)
Comment (Ambrosegithub) [reply index.html]: DONE

File: index.html (hunk 5 above)
Comment (Ambrosegithub) [reply index.html]: DONE

**Comment by Ambrosegithub** [comment]
DONE

File: index.html (hunk 6 above)
Comment (Ambrosegithub) [reply index.html]: done

**Review by Kossi-stack — APPROVED** [review]
## Status: Approved ✔️
**Hi @Ambrosegithub,**

You did a really great job! 💯 💯💯💯
Congratulations! 🎉 🎉 🎉 🎉 🎉


You may close this Pull Request and merge the changes
Keep being awesome dev's and good luck with your next milestone!

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding! 👏 👏 👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Pull request merged**